make/tsb.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
let's re-use as much as we can of https://github.com/tetrateio/tetrate-service-express-sandbox/blob/main/make/tetrate_managementplane.sh and have a separate helper for controlplane https://github.com/tetrateio/tetrate-service-express-sandbox/blob/main/make/tetrate_controlplane.sh